test/e2e/app-dir/ppr-navigations/simple/per-segment-prefetching.test.ts

```diff
@@ -36,32 +36,24 @@ describe('per segment prefetching', () => {
     return null
   }
 
-  it('basic prefetching flow', async () => {
+  it('basic route tree prefetch', async () => {
     // To perform a prefetch a page, the client first fetches the route tree.
     // The response is used to construct prefetches of individual segments.
     const routeTreeResponse = await prefetch('/en', '/_tree')
     const routeTreeResponseText = await routeTreeResponse.text()
     const routeTree = extractPseudoJSONFromFlightResponse(routeTreeResponseText)
 
+    // Confirm that the prefetch was successful. This is a basic check to ensure
+    // that the name of an expected field is present in the response.
+    expect(typeof routeTree.staleTime).toBe('number')
+
     // The root segment is a shared segment. Demonstrate that fetching the root
     // segment for two different pages results in the same response.
     const enResponse = await prefetch('/en', '/')
     const enResponseText = await enResponse.text()
     const frResponse = await prefetch('/fr', '/')
     const frResponseText = await frResponse.text()
     expect(enResponseText).toEqual(frResponseText)
-
-    // Now use the route tree to construct a request for the child segment.
-    const child = routeTree.tree.slots.children
-
-    // The access token is appended to the end of the segment path.
-    const fullChildSegmentPath = `${child.path}.${child.token}`
-    const childResponse = await prefetch('/en', fullChildSegmentPath)
-    const childResponseText = await childResponse.text()
-
-    // Confirm that the prefetch was successful. This is a basic check to ensure
-    // that the name of an expected field is somewhere in the Flight stream.
-    expect(childResponseText).toInclude('"rsc"')
```

Comment:
how come we no longer assert on this behavior?

Reply:
So that it's not coupled to the cache key generation algorithm, which happened to change in this PPR. Really we could just delete this whole test. The original test was written before any of the client behavior was implemented; at this point it's more than covered by the e2e tests.